gnucash/import-export/aqb/gnc-gwen-gui.c

```diff
@@ -1072,26 +1072,18 @@ get_input(GncGWENGui *gui, guint32 flags, const gchar *title,
 
         if(error != NULL)
         {
-            printf("Error; perhaps mimeType (%s) not supported?\n%s\n", mimeType, error->message);
+            PERR("Pixbuf loader not loaded: %s, perhaps MIME type %s isn't supported.", error->message, mimeType);
         }
 
-        if(gdk_pixbuf_loader_write(loader, gudata, lChallenge, &error) == 0)
-        {
-            printf("Error:\n%s\n", error->message);
-        }
-
-        if(gdk_pixbuf_loader_close(loader, &error) == 0)
-        {
-            printf("Error:\n%s\n", error->message);
-        }
+        gdk_pixbuf_loader_write(loader, gudata, lChallenge, &error);
```

Comment:
If you're going to ignore the error pass `NULL` instead of &error, but it's probably better to log the errors with PERR or PWARN. 

As an aside for other programs: Normal practice is that the default stream for output, `stdout`, is used for the ordinary output of a command-line program, and `stderr` is used for exception output. See [printf(3)](https://linux.die.net/man/3/printf) and note `fprintf`. GUI programs may or may not have `stdout` and `stderr` defined so you need to look for (or write, if it's your own project) a logging facility. GnuCash's is in `libgnucash/engine/qoflog.h`.

Reply:
Okay, I will go the easy way here and just pass NULL for the second parameter. I originally inserted the printfs as a quick and dirty debugging means.